packages/evolution-frontend/src/components/inputs/maps/google/GoogleGeocoder.tsx

```diff
@@ -61,8 +62,14 @@ export const geocodeMultiplePlaces = (
             { lat: options.bbox[0], lng: options.bbox[1] },
             { lat: options.bbox[2], lng: options.bbox[3] }
         );
+
+        const upperBound = turfPoint([options.bbox[0], options.bbox[1]]);
+        const lowerBound = turfPoint([options.bbox[2], options.bbox[3]]);
+        const viewportRadius = turfDistance(upperBound, lowerBound, { units: 'meters' }) / 2;
+        const searchRadius = Math.min(Math.max(viewportRadius, 500), 8000);
```

Comment:
There are meters? Could you extract the 500 and 8000 in constants with a meaningful name